demos/text_detection_demo/src/text_recognition.cpp

```diff
@@ -53,3 +75,81 @@ std::string CTCGreedyDecoder(const std::vector<float> &data, const std::string&
     }
     return res;
 }
+
+std::string CTCBeamSearchDecoder(const std::vector<float> &data, const std::string& alphabet, char pad_symbol, double *conf, int bandwidth) {
+    std::string res = "";
```

Comment:
`res` is not used until the very end of the function, so it would be better for readability to move this declaration to the bottom. Also, the initializer is redundant.